app/components/post_list/post/avatar/index.tsx

```diff
@@ -44,8 +44,7 @@ const style = StyleSheet.create({
     },
     profilePictureContainer: {
         marginBottom: 5,
-        marginLeft: 12,
-        marginRight: Platform.select({android: 11, ios: 10}),
+        marginRight: 10,
         marginTop: 10,
```

Comment:
should we keep top/bottom margins?

Reply:
this is basically the container for the profilePicture int the post component, so I see no reason why not, but it could be moved to the post component but that would mean to add another View just for that, which has more cost.

Reply:
@koox00 thoughts?